appcenter-data/Data.js

```diff
@@ -19,13 +19,34 @@ const DefaultPartitions = {
 const Data = {
     TimeToLive,
     DefaultPartitions,
+    isEnabled,
```

Comment:
Do we have the typescript for those?

Reply:
No, I plan on adding TypeScript declarations in a separate PR.

Reply:
@jackycodes Track a new PR and you can resolve this :)